09-workload.md

```diff
@@ -28,18 +28,24 @@ The cluster now has an [Traefik configured with a TLS certificate](./08-secret-m
    kubectl get ingress aspnetapp-ingress -n a0008
    ```
 
-   > At this point, the route to the workload is established, SSL offloading configured, and a network policy is in place to only allow Traefik to connect to your workload. Therefore, you should expect a `403` HTTP response if you attempt to connect to it directly.
+   > At this point, the route to the workload is established, SSL offloading configured, a network policy is in place to only only allow Traefik to connect to your workload, and Traefik is configured to only accept requests from App Gateway.
```

Comment:
"only" is twice in the sentence.

"to only only allow"

Reply:
dang dang